examples/sudoku/ts/src/sudoku.test.ts

```diff
@@ -17,45 +18,30 @@ describe('sudoku', () => {
 
   beforeEach(async () => {
     await isReady;
-    account = createLocalBlockchain();
+    let Local = Mina.LocalBlockchain({ proofsEnabled: false });
+    Mina.setActiveInstance(Local);
+    account = Local.testAccounts[0].privateKey;
     zkAppPrivateKey = PrivateKey.random();
     zkAppAddress = zkAppPrivateKey.toPublicKey();
     zkAppInstance = new SudokuZkApp(zkAppAddress);
```

Comment:
We should rename this to `zkApp` to be consistent with your rename in the `run.ts` file, or change the name to `zkAppInstance` in `run.ts`. I'm fine with either name as long as it is consistent across files.